.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
```

Comment:
Probably above this should be a link [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md)

Reply:
I would -1 for linking for the reasons @DevinWalker has said.